src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs

```diff
@@ -272,5 +278,47 @@ public void Dispose()
             clientData.pSignerParams->pSignCallBack = signInfo;
 
         }
-    }
+
+        /// <summary>
+        /// It is required to call initialize before using the AzureSign to ensure dll's are loaded correctly.
+        /// The initialize should happen as early as possible in the host process.
+        /// </summary>
+        /// <param name="assemblyPath">Override the default location for the assembly. Necessary when running on Azure App Services</param>
+        public static void Initialize(string assemblyPath = null)
+        {
+            if (ManifestLocation != null)
+                return; // already initialized
+
+
+            var is64bit = IntPtr.Size == 8;
+
+            if(!string.IsNullOrWhiteSpace(assemblyPath))
```

Comment:
Did you mean to assign this if it _is_ null or whitespace?

Reply:
Yeah, fixed.